src/components/Dashboard/Header/Authentication/Signup.jsx

```diff
@@ -0,0 +1,163 @@
+import React, { Component } from 'react';
+import {
+  Form, Button, Container, Row, Col,
+} from 'react-bootstrap';
+import { Redirect } from 'react-router-dom';
+import { Toaster, Intent } from '@blueprintjs/core';
+import db from '../../../../fire';
+import NavBar from '../Header';
+
+export default class Signup extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      redirect: false,
```

Comment:
it would be more understandable if you rename `redirect `into `isLogedIn/isUserLogedIn`